scripts/delayed.js

```diff
@@ -4,4 +4,20 @@ import { loadScript, sampleRUM } from './aem.js';
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
+// This function is called when the Turnstile script is loaded and ready to be used.
+// The function name matches the "onload=..." parameter.
+// eslint-disable-next-line no-unused-vars
+function turnstileCb() {
+  document.querySelectorAll('.turnstile-container').forEach((el) => {
+    // eslint-disable-next-line no-undef
+    turnstile.render(el, {
+      sitekey: '0x4AAAAAAAVrq_k7QabpK5gM', // TODO: Replace with actual sitekey
```

Comment:
snippet from current website
`
    var validateuserToken = "";


    window.onloadTurnstileCallback = function () {
        turnstile.render('.captcha-container', {
            sitekey: '0x4AAAAAAAK3iakV3QAmUWAf',
            callback: function (token) {
                validateuserToken = token;
                console.log(`Challenge Success ${token}`);
            },
        });
        turnstile.render('#captcha-container1', {
            sitekey: '0x4AAAAAAAK3iakV3QAmUWAf',
            callback: function (token) {
                validateuserToken = token;
                console.log(`Challenge Success ${token}`);
            },
        });
    }
`
Website seems to store the token in validateUserToken and use it later to check if user has been validated or not. 
IMO we should include that as well